lib/active_model/serializer/reflection.rb

```diff
@@ -62,11 +62,12 @@ def build_association(subject, parent_serializer_options)
 
       private
 
-      def serializer_options(parent_serializer_options, reflection_options)
+      def serializer_options(parent_serializer_options, reflection_options, subject)
         serializer = reflection_options.fetch(:serializer, nil)
```

Comment:
If you feel strongly about this, and think of a better name, I don't mind changing it.